run.sh

```diff
@@ -14,5 +14,5 @@ migrate)
     bundle exec rake db:schema:load
     ;;
 esac
-bundle exec import:all assets:precompile
+RAILS_ENV=production bundle exec import:all assets:precompile
```

Comment:
As I said previously, I don't think it makes sense to change just this line. `RAILS_ENV` should be set to `production` for every `bundle exec …` command, so it should be set once somewhere that covers all these command lines and the servers.